pkg/kapis/admission/v1alpha1/register.go

```diff
@@ -0,0 +1,137 @@
+/*
+Copyright 2021 KubeSphere Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	"github.com/emicklei/go-restful"
+	restfulspec "github.com/emicklei/go-restful-openapi"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	ksapi "kubesphere.io/kubesphere/pkg/api"
+	v1alpha1 "kubesphere.io/kubesphere/pkg/api/admission/v1alpha1"
+	"kubesphere.io/kubesphere/pkg/apiserver/runtime"
+	"kubesphere.io/kubesphere/pkg/constants"
+	"kubesphere.io/kubesphere/pkg/informers"
+	"kubesphere.io/kubesphere/pkg/simple/client/admission"
+	"net/http"
+)
+
+const (
+	GroupName = "admission.kubesphere.io"
+)
+
+var GroupVersion = schema.GroupVersion{Group: GroupName, Version: "v1alpha1"}
+
+func AddToContainer(container *restful.Container, informers informers.InformerFactory, option *admission.Options) error {
+	ws := runtime.NewWebService(GroupVersion)
+	handler := newAdmissionHandler(informers, option)
+
+	// List
+	ws.Route(ws.GET("/policyTemplates").
```

Comment:
```suggestion
	ws.Route(ws.GET("/policytemplates").
```

Uppercase is unnecessary.